spec/lib/response-handler.spec.js

```diff
@@ -0,0 +1,128 @@
+"use strict";
+
+describe("lib/response-handler", () => {
+    var bluebird, error, logger, parseLinkHeader, responseHandler, URI;
+
+    bluebird = require("bluebird");
+    error = require("../../lib/error")();
+    logger = require("../../lib/logger")();
+    parseLinkHeader = require("parse-link-header");
+    URI = require("urijs");
+    responseHandler = require("../../lib/response-handler")(bluebird, error, logger, parseLinkHeader, URI);
+
+    describe(".isErrorCode()", () => {
```

Comment:
Could you test like 200 and 201 as well? Those are actual real world codes that must succeed.